plugins/Installation/templates/finished.twig

```diff
@@ -27,6 +27,10 @@
         </p>
     {% endif %}
 
+    <h3>Performance Settings</h3>
+    <p>{{ 'Installation_PerformanceSettingsDesc1'|translate('<a href="https://matomo.org/docs/setup-auto-archiving/" rel="noreferrer noopener">', '</a>')|raw }}</p>
```

Comment:
could we add a target _blank here for both links @diosmosis ? That be great as when I clicked on it I left the installer